src/main/java/com/hladchenko/springrestapi/controller/UserController.java

```diff
@@ -0,0 +1,60 @@
+package com.hladchenko.springrestapi.controller;
+
+import com.hladchenko.springrestapi.entity.Note;
+import com.hladchenko.springrestapi.entity.User;
+import com.hladchenko.springrestapi.service.UserService;
+import org.springframework.hateoas.EntityModel;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.UUID;
+
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.linkTo;
+import static org.springframework.hateoas.server.mvc.WebMvcLinkBuilder.methodOn;
+
+@RequestMapping("/user")
+@RestController
+public class UserController {
+    UserService userService;
+
+    public UserController(UserService userService) {
+        this.userService = userService;
+    }
+
+    @PostMapping
+    public EntityModel<User> createUser(@RequestBody User user) throws Exception {
```

Comment:
why some of the methods are marked that they can throw an exception? 
is it necessary?

Reply:
Nice catch! It is left from the previous code.